tests/runtime_neutral/test_vibe_specialist_consultation.py

```
@@ -959,6 +1076,59 @@ def test_host_user_briefing_segment_handles_missing_consultation_receipt(self) -
+                    )
+                }
+            }
+            $result = New-VibeHostUserBriefingProjection -DeliveryAcceptanceReport $deliveryAcceptanceReport
+            $result | ConvertTo-Json -Depth 20
+            """
+        )
+
+        self.assertEqual("execution_handoff_host_briefing", result["mode"])
+        self.assertEqual(["execution_handoff"], [str(segment["segment_id"]) for segment in list(result["segments"])])
+        self.assertIn("Execution handoff is still pending under governed vibe.", result["rendered_text"])
+        segment = list(result["segments"])[0]
```

Ruff (0.15.11)

[warning] Prefer `next(iter(result["segments"]))` over single element slice

Replace with `next(iter(result["segments"]))`

(RUF015)